ctru-sys/src/lib.rs

```diff
@@ -16,6 +16,24 @@
 pub mod result;
 pub use result::*;
 
+// Fun fact: C compilers are allowed to represent enums as the smallest integer type that can hold all of its variants,
+// meaning that enums are allowed to be the size of a `c_short` or a `c_char` rather than the size of a `c_int`.
+// Libctru's `errorConf` struct contains two enums that depend on this narrowing property for size and alignment purposes,
+// and since `bindgen` generates all enums with `c_int` sizing, we have to blocklist those types and manually define them
+// here with the proper size.
```

Comment:
This is really interesting! Similar to what I mentioned in #166 , maybe we can just force these constants to generate using `errorReturnCode` as the type instead of `c_int`?

To catch the struct size/alignment issues, I wonder if we could have caught that using bindgen's layout tests. I think they were omitted before because we didn't have any good way to run them, but with `test-runner` now we might be able to actually do it in CI. I'll file an issue